src/app/[locale]/globals.css

```
@@ -7,7 +7,89 @@
 		text-wrap: balance;
 	}
 
+	/* 
+	===============
+	Extra Heading 
+	===============
+	*/
 	.vm-text-eh1 {
 		@apply text-6xl font-semibold leading-[70px] tracking-tight;
 	}
+	.vm-text-eh2 {
+		@apply text-4xl font-semibold leading-[53.2px] tracking-tight;
+	}
+	.vm-text-eh3 {
+		@apply text-3xl font-semibold leading-[44.8px] tracking-tight;
+	}
+	.vm-text-eh4 {
+		@apply text-[28px] font-semibold leading-[39.2px] tracking-tight;
```

No need to be pixel perfect. Let's go for text-3xl for this one as well.

```suggestion
		@apply text-3xl font-semibold leading-[39.2px] tracking-tight;
```